stock_valuation_layer_total_value/models/stock_valuation_layer.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class StockValuationLayer(models.Model):
+    _inherit = "stock.valuation.layer"
+
+    original_value = fields.Float(
+        string="Original value",
+        compute="_compute_original_layer_values",
+    )
+    adjusted_unit_price = fields.Float(
```

Comment:
I don't see how a field called "adjusted_unit_price" shows as string "New Value", should it be "new unit price"?

Reply:
@AaronHForgeFlow   I changed the name